notebooks/ai-ml/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.      model = resnet50(ResNet50_Weights.DEFAULT)
</pre><p>Here, we are no longer broadcasting the model weights which was done earlier, so now they will be downloaded parallely for each partition? It would show up in executor logs not on the notebook if downloading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
I now load the model on the driver and it appears Spark can successfully broadcast.